lib/pbench/test/unit/agent/task/test_results_push.py

```diff
@@ -17,26 +18,20 @@ class TestResultsPush:
     URL = "http://pbench.example.com/api/v1"
 
     @staticmethod
-    def add_success_mock_response():
+    def add_http_mock_response(code: int = 200):
```

Comment:
Do we like the literal `200` here, or should we be using a symbolic reference?

Reply:
Ditto line 180.

Reply:
Moved to #2287.